modules/hcp-virt-taints-tolerations.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-virt-taints-tolerations_{context}"]
+= Custom taints and tolerations
+
+For {hcp} on {VirtProductName}, by default, pods for a hosted cluster tolerate the `control-plane` and `cluster` taints. However, you can also use custom taints on nodes so that hosted clusters can tolerate those taints on a per-hosted-cluster basis by setting `HostedCluster.spec.tolerations`.
+
+:FeatureName: Passing tolerations for a hosted cluster
+include::snippets/technology-preview.adoc[]
+
+.Example configuration
+[source,yaml]
+----
+  spec:
+    tolerations:
+    - effect: NoSchedule
+      key: kubernetes.io/custom
+      operator: Exists
+----
+
+You can also set tolerations on the hosted cluster while you create a cluster by using the `--tolerations` hcp CLI argument.
+
+.Example CLI argument
+[source,terminal]
+----
+--toleration="key=kubernetes.io/custom,operator=Exists,effect=NoSchedule"
+----
+
+For fine granular control of hosted cluster pod placement on a per-hosted-cluster basis, use custom tolerations with `nodeSelectors`. You can co-locate groups of hosted clusters and isolate them from other hosted clusters. You can also place hosted clusters in infra and master nodes.
+
+Tolerations on the hosted cluster spread only to the pods of the control plane. To configure other pods that run on the management cluster and infrastructure-related pods, such as the pods to run VMs, you need to use a different process.
```

Comment:
Slight suggestion - although VM is sort of well defined already and plenty of people know what it means, I would recommend spelling it out on its first instance in a section:
```suggestion
Tolerations on the hosted cluster spread only to the pods of the control plane. To configure other pods that run on the management cluster and infrastructure-related pods, such as the pods to run virtual machines, you need to use a different process.
```

Reply:
Fixed